origami.json

```diff
@@ -9,19 +8,41 @@
 		"slack": "financialtimes/#origami-support"
 	},
 	"supportStatus": "experimental",
-	"browserFeatures": {},
+	"browserFeatures": {
+		"required": [
+			"String.prototype.startsWith"
+		]
+	},
 	"demosDefaults": {
-		"sass": "demos/src/demo.scss",
-		"js": "demos/src/demo.js",
+		"sass": "demos/src/shared.scss",
 		"documentClasses": "",
-		"dependencies": []
+		"dependencies": [
+			"o-forms@9.0.0-beta.1",
+			"o-normalise@3.0.0-beta.0",
+			"o-utils@2.0.0-beta.0"
```

Comment:
```suggestion
			"o-forms@^9.0.0",
			"o-normalise@^3.0.0",
			"o-utils@^2.0.0"
```

Reply:
Done in [`49b0363` (#2)](https://github.com/Financial-Times/o-autocomplete/pull/2/commits/49b0363ed2605eeb4fbcb4159256fdd546ab0680)